dungeon/src/core/level/DungeonLevel.java

```diff
@@ -24,8 +25,11 @@
  *
  * @see core.level.elements.ILevel
  */
-public class TileLevel implements ILevel, ITickable {
+public class DungeonLevel implements ILevel, ITickable {
 
+  protected final List<Coordinate> customPoints = new ArrayList<>();
+  protected String levelName = "Default Level Name";
```

Comment:
Wir sollten kein Default Namen haben. Ich würde sowas evtl. vorschlagen:

```suggestion
  private static int levelNameSuffix = 1;
  protected String levelName;
  
  // Im Ctor:
  levelName = "level_" + levelNameSuffix++;
```